contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,153 @@
+
+The database is linked right into PostHog, making it easy to see associated user and organization profiles. It’s also hooked into Vitaly, our customer success tool, to automatically pull in more customer and business information.
+
+HERMES also uses ChatGPT to generate a summary of the features requested during the interview based on the human-authored interview notes. This makes it easy to share actionable feedback from users directly with the product team responsible for that feature. 
+
+As part of the project, we also revamped our system for categorizing and tracking feature requests from customers, making it easier for us to see what people want and prioritize the most important product work.
+
+### Data crunching
+
+<BuiltBy people={["Sandy Spicer", "Tim Glaser"]} />
+
+PostHog crunches a lot of data – and sometimes queries can take a while to load. To help users better understand the hard work we’re doing when they make a query, we built a loading bar that shows statistics on the data we’re crunching, and the CPU we’re throwing at each query. The loading bar is fake right now (sorry!) but the data we show is very much real.
+
+
+### VIP customer lookup
+
+<BuiltBy people={["Simon Fisher"]} />
+
+When a customer reports a problem, we want our engineers to get it fixed as soon as possible – but right now our engineers lack context on which customers to prioritize. Right now, they ping our customer success team on Slack for details on accounts. This isn’t good practice in an [async company](/newsletter/how-we-work-async), especially when this information is already written down somewhere.
+
+Enter the VIP Customer Lookup bot. Ping this clever little Slackbot with any customer ID and it’ll return key details of any account, including the plan they are on, how much money they spend, and who on customer success looks after the account.
+
+### CLI
+
+<BuiltBy people={["Manoel Aranda Neto", "Daniel Esneider"]} />
+
+We build products for engineers, so there’s nothing better than bringing PostHog closer to their natural environment: the terminal.
+
+The PostHog CLI is a command line that allows users to do a few things that are normally tucked away in the PostHog app: creating, reading, updating, deleting, and enabling or disabling feature flags, for example. The PostHog CLI authentication flow is also seamless as it spins up a new browser and allows you to log in with your SSO instead of copying and pasting tokens manually.
+
+In the future, the CLI could be expanded with more features such as creating surveys, and events, installing SDKs automatically, uploading debug symbols, using the CLI as a package, or even as a GitHub action. And, even better, you could do all of that with natural language, no need to memorize all commands by heart.
```

Comment:
Is this intended for internal use only @marandaneto?

Reply:
yes, it depends on backend change and we didn't roll it out because we need to check the "security side of things" first, Ben was gonna check this but not yet.

Reply:
@andyvan-ph lemme know what you think is best here and i can update the copy

Reply:
We can just say it's internal for now, but could become public in future.

Also, would be good to have a screengrab for this.